configuration/pih/reports/sql/covid19encounterexport.sql

```diff
@@ -6,98 +6,1328 @@
+SET 
+    crp = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '161500');
+
+UPDATE temp_encounter
+SET 
+    sodium = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '1132');
+
+UPDATE temp_encounter
+SET 
+    potassium = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '1133');
+
+UPDATE temp_encounter
+SET 
+    urea = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '857');
+
+UPDATE temp_encounter
+SET 
+    creatinine = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '790');
+
+UPDATE temp_encounter
+SET 
+    glucose = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '887');
+
+UPDATE temp_encounter
+SET 
+    bilirubin = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '655');
+
+UPDATE temp_encounter
+SET 
+    sgpt = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '654');
+
+UPDATE temp_encounter
+SET 
+    sgot = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '653');
+
+UPDATE temp_encounter
+SET 
+    pH = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '165984');
+
+UPDATE temp_encounter
+SET 
+    pcO2 = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '163595');
+
+
+UPDATE temp_encounter
+SET 
+    pO2 = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '163598');
+
+UPDATE temp_encounter
+SET 
+    tcO2 = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '166002');
+
+UPDATE temp_encounter
+SET 
+    hcO3 = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '163596');
+
+UPDATE temp_encounter
+SET 
+    be = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '163599');
+
+UPDATE temp_encounter
+SET 
+    sO2 = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '163597');
+UPDATE temp_encounter 
+SET 
+    lactate = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '165997');
+
+#######Radiology
+UPDATE temp_encounter 
+SET 
+    chest_xray = OBS_SINGLE_VALUE_CODED(encounter_id,
+            'PIH',
+            'Radiology procedure performed',
+            'CIEL',
+            '165152');
+
+UPDATE temp_encounter e
+        LEFT JOIN
+    (SELECT 
+        encounter_id, person_id, obs_group_id, value_text
+    FROM
+        obs
+    WHERE
+        obs_group_id IN (SELECT 
+                obs_group_id
+            FROM
+                obs o
+            WHERE
+                voided = 0
+                    AND concept_id = CONCEPT_FROM_MAPPING('PIH', 'Radiology procedure performed')
+                    AND value_coded = CONCEPT_FROM_MAPPING('CIEL', '165152')
+                    AND encounter_id IN (SELECT 
+                        encounter_id
+                    FROM
+                        encounter
+                    WHERE
+                        encounter_type = @covid_admission_encounter_type))
+            AND concept_id = CONCEPT_FROM_MAPPING('PIH', 'Radiology report comments')
+            AND voided = 0) o1 ON e.patient_id = o1.person_id
+        AND e.encounter_id = o1.encounter_id 
+SET 
+    chest_xray_findings = o1.value_text;
+
+UPDATE temp_encounter 
+SET 
+    cardioUS = OBS_SINGLE_VALUE_CODED(encounter_id,
+            'PIH',
+            'Radiology procedure performed',
+            'CIEL',
+            '163041');
+
+UPDATE temp_encounter e
```

Comment:
Same here.  This complicated update, run multiple times, needs to be pulled into a function.